app/src/main/res/navigation/nav_graph.xml

```diff
@@ -335,6 +335,11 @@
             app:destination="@id/bookmarkSearchDialogFragment"
             app:popUpTo="@id/bookmarkSearchDialogFragment"
             app:popUpToInclusive="true" />
+         <action
+            android:id="@+id/action_global_search_dialog"
```

Comment:
Just noticed this. Why do we need to add an action here if this is already global? https://searchfox.org/mozilla-mobile/source/fenix/app/src/main/res/navigation/nav_graph.xml#37

Otherwise, we should follow the id convention - `action_bookmarkFragment_to_searchDialogFragment`. However, I don't think we need to add this action and just call the global action. We will probably also want to fix this for the history fragment as well.

Reply:
You're probably right.  I'll check.  Thanks

Reply:
Removed.  Good catch, thanks